keycloak/keycloak-admin-ui
# Add text for user federation user attribute mapper

## Motivation
Prepares the text for the forms within this modal https://marvelapp.com/prototype/ecjc94b/screen/72071054
Note that selecting mapper type changes what fields are in the form.
Also deduplicates text strings used in multiple places: name, role, category, type, priority, description

## Brief Description
Adds text to user fed messages and help files as well as common messages file. Moves some definitions that were in multiple places to common-messages.

## Verification Steps
New text for user fed mapper dialog can be checked by comparing to the text when adding/editing a new mapping in the old interface.
Consolidation of duplicate definitions for name, role, category, type, priority, and description should be checked anywhere they are used. Check through screens that none were missed. (which would show as lower case if the definition was removed without pointing to common now)

## Checklist:

- [ x] Code has been tested locally by PR requester
- [x ] User-visible strings are using the react-i18next framework (useTranslation)
- [ x] Help has been implemented
- [ na] axe report has been run and resulting a11y issues have been resolved
- [na ] Unit tests have been created/updated
- [ x] Formatting has been performed via prettier/eslint
- [ x] Type checking has been performed via 'yarn check-types'
